onnxruntime/contrib_ops/cpu/transformers/logits_processor.h

```diff
@@ -150,19 +159,25 @@ class PresencePenaltyLogitsProcessor : public ILogitsProcessor<T> {
 template <typename T>
 class TimestampLogitsProcessor : public ILogitsProcessor<T> {
  public:
-  TimestampLogitsProcessor(int eos_token_id, int max_initial_timestamp_index)
-      : eos_token_id_(eos_token_id), max_initial_timestamp_index_(max_initial_timestamp_index) {}
+  TimestampLogitsProcessor(int eos_token_id,         // <|endoftext|>
+                           int sot_token_id,         // <|startoftranscript|>
+                           int translate_token_id,   // <|translate|>
+                           int transcribe_token_id,  // <|transcribe|>
+                           int solm_token_id,        // <|startoflm|>
```

Comment:
please not use abbreviations here since those (solm, not, and beg) are not well-known ones.

Reply:
These abbreviations are from [this PR](https://github.com/microsoft/onnxruntime/pull/15853). I have fixed them now.